docs/sources/tempo/setup/upgrade.md

```diff
@@ -15,6 +15,41 @@ This upgrade guide applies to on-premise installations and not for Grafana Cloud
 
 >**TIP**: You can check your configuration options using the [`status` API endpoint]({{< relref "../api_docs#status" >}}) in your Tempo installation.
 
+## Upgrade to Tempo 2.2
+
+Tempo 2.2 has several considerations for any upgrade:
+
+* vParquet2 is now the default block format
+* Several configuration parameters have been renamed or removed.
+
+For a complete list of changes, enhancements, and bug fixes, refer to the [Tempo 2.2 changelog](https://github.com/grafana/tempo/releases).
+
+### Default block format changed to vParquet2
+
+While not a breaking change, upgrading to Tempo 2.2 will by default change Tempo’s block format to vParquet2.
+
+To stay on a previous block format, read the[Parquet configuration documentation]({{< relref "../configuration/parquet#choose-a-different-block-format" >}}).
+We strongly encourage upgrading to vParquet2 as soon as possible as this is required for using structural operators in your TraceQL queries and provides query performance improvements, in particular on queries using the `duration` intrinsic.
+
+### Updated JSonnet supports `statefulset` for the metrics-generator
+
+Tempo 2.2 updates the `microservices` JSonnet to support a `statefulset` for the `metrics_generator` component.
+
+To support a new `processor`, the metrics-generator has been converted from a `deployment` into a `statefulset` with a PVC.
```

Comment:
may be worth noting this is only important if you use the `local-blocks` processor which is experimental and most people won't use